Add alt-d shortcut to select address bar text

MichaelMcDonnell opened the pull request:
I usually use the alt-d in shortcut in Firefox to select the text in the address bar. Firefox supports both alt-d and ctrl-l. This change adds the alt-d shortcut so you can use either.

Testing: I tested it by running Servo and pressing alt-d. I don't know if there is a way to create a test for it?

MichaelMcDonnell committed:
Add alt-d shortcut to select address bar text

I usually use the alt-d in shortcut in Firefox to select the text in the
address bar. Firefox supports both alt-d and ctrl-l. This change adds
the alt-d shortcut so you can use either.

Signed-off-by: Michael Mc Donnell <michael@mcdonnell.dk>

mukilan reviewed (commented):
On Mac, with this patch, Option+D focuses the address bar but that doesn't seem to the case in FF or Safari. Not a big deal, as this is just our "demo" browser, but maybe worth checking if this is avoidable.

MichaelMcDonnell committed:
Only support alt-d on Linux and Windows

Firefox on macOS only supports command-l and not alt-d. The alt-d
shortcut should only work on Linux and Windows.

Signed-off-by: Michael Mc Donnell <michael@mcdonnell.dk>

MichaelMcDonnell commented:
Thanks for merging it! 😀
